src/pages/ApartmentShow.js

```diff
@@ -1,11 +1,23 @@
 import React from "react"
+import { useParams } from "react-router-dom"
 
+const ApartmentShow = ({ apartments }) => {
+  const { id } = useParams()
+  let currentApartment = apartments?.find((apartment) => apartment.id === +id)
 
-const ApartmentShow = () => {
   return (
-    <div>
-      {/* <h3>ApartmentShow</h3> */}
-    </div>
+    <main className="apartment-show-cards">
+      {currentApartment && (
+        <>
+         <img
+            alt={`Preview of an apartment ${currentApartment.unit}`}
+            src={currentApartment.image}
+            className="apartment-show-img"
+          />
+          {/* additional apartment code here */}
```

Comment:
If you plan on displaying additional apartment code, this is the time to implement it.